### ISSUE TITLE
`::fragment:nth-child(1)` should work?

### ISSUE THREAD

**Issue by argyleink** [flackr/carousel#18]

**Comment by flackr** [comment]
Do you expect this to select the first fragment? If so, I think we should add the parameter to the fragment function, e.g. `::fragment(1)` or `::nth-fragment(1)`. The fragments are not proper "children" so I'm not sure if it's sensible to use nth-child here.

**Comment by argyleink** [comment]
yes, select the first fragment. not proper children.. but children nonetheless yeah?

what about `::fragment:snapped`? view transitions precedence here might be useful to use for influence?
